tests/modeltests/tst_disassemblyoutput.cpp

```diff
@@ -189,6 +190,80 @@ private slots:
         QCOMPARE(findSourceCodeFile(QStringLiteral("./liba/lib.c"), {tempDir.path()}, QString()),
                  tempDir.path() + QDir::separator() + QStringLiteral("liba/lib.c"));
     }
+
+    void testDetectBranches()
+    {
+        if (!supportsVisualizeJumps()) {
+            QSKIP("--visualize-jumps is not supported");
+        }
+
+        const auto lib = QFileInfo(findLib(QStringLiteral("libfib.so")));
+        auto [address, size] = findAddressAndSizeOfFunc(lib.absoluteFilePath(), QStringLiteral("_Z3fibi"));
+
+        const Data::Symbol symbol = {QStringLiteral("fib(int)"), address, size, QStringLiteral("libfib.so")};
+
+        const QString objdump = QStandardPaths::findExecutable(QStringLiteral("objdump"));
+        const auto result =
+            DisassemblyOutput::disassemble(objdump, {}, QStringList {lib.absolutePath()}, {}, {}, {}, symbol);
+        QVERIFY(result.errorMessage.isEmpty());
+
+        auto isHex = [](const QChar c) {
+            return (c >= QLatin1Char('0') && c < QLatin1Char('9')) || (c >= QLatin1Char('a') && c <= QLatin1Char('f'));
+        };
+
+        for (const auto& line : result.disassemblyLines) {
+            QVERIFY(!line.branchVisualisation.isEmpty());
+            if (!line.disassembly.isEmpty()) {
+                QVERIFY(isHex(line.disassembly[0]));
```

Comment:
```
bool ok = false;
auto addr = line.disassembly[0].toULongLong(&ok, 16);
QVERIFY(ok);
QVERIFY(addr > 0);
```

and then remove `isHex`

Reply:
can we also test then that the `addr` is in the range we got from `findAddressAndSizeOfFunc`?

Reply:
its not the address, this is the hexdump, but yes will do